### ISSUE TITLE
Unable to set querystring

### ISSUE THREAD

**Issue by zhonghao-cliqz** [cliqz/url-parser#22]
For the following snippet,
```javascript
let url = new URL('https://github.com');
url.searchParams.set('key', 'value');
console.log(url.href);
```
Standard `URL` returns `https://github.com/?key=value` while `url-parser` gives me `https://github.com`.

If this is out of the scope of this library (I guess is being a super fast parser), feel free to close the issue.

**Comment by sammacbeth** [comment]
Changes to search params are currently not propagated to the `URL` object they are in. Ideally this library should be compatible with the standard `URL`, so we should fix this.

**Comment by sammacbeth** [comment]
Fixed in `v1.1.1`
